### PR TITLE
Preparing storage for beta release

### PR DESCRIPTION
**Pull request by gapra-msft**

### CODE REVIEW

**Review by alzimmermsft — APPROVED** [review]

File: sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/options/FileSystemUndeleteOptions.java [hunk 1]
```diff
@@ -46,31 +46,5 @@ public String getDeletedFileSystemVersion() {
...
-     * already exists, the undelete operation will result in a 409 (conflict).
-     *
-     * @param destinationFileSystemName The destination file system name.
-     * @return The updated options.
-     */
-    public FileSystemUndeleteOptions setDestinationFileSystemName(String destinationFileSystemName) {
-        this.destinationFileSystemName = destinationFileSystemName;
-        return this;
-    }
+    // TODO (gapra) : Bring back destination file system name when there is support for it
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/options/FileSystemUndeleteOptions.java:49]: I'd just get rid of this and tests. It's not worth cluttering codebase if timeline is unknown. Better create gh issue.

**Review by kasobol-msft — APPROVED** [review]

**Review by seanmcc-msft — APPROVED** [review]

**Review by jaschrep-msft — APPROVED** [review]

File: sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/options/FileSystemUndeleteOptions.java (hunk 1 above)
Comment (gapra-msft) [reply sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/options/FileSystemUndeleteOptions.java]: https://github.com/Azure/azure-sdk-for-java/issues/21353

Comment (gapra-msft) [reply sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/options/FileSystemUndeleteOptions.java]: Also removed the code

**Pull request merged**